.travis.yml

```diff
@@ -0,0 +1,11 @@
+language: python
+python:
+  - "2.7"
+  - "3.6"
```

Comment:
There's no point in testing with two Python versions, since Python is only required for the tools we're using, we're not testing any Python code here?

I'd stick to Python 3.6 if that works, more future proof...